assets/styles/landing.scss

```diff
@@ -28,6 +29,43 @@
     width: 100%;
 }
 
+.features {
+    align-items: center;
+    background: $gallery;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: space-around;
+    position: relative;
+
+    .feature {
+        color: $dusty-gray;
+        font-weight: bold;
+        max-width: 250px;
+        text-align: center;
+        width: 100%;
+
+        .img {
+            margin: 40px 35px 0;
+        }
+
+        .title {
+            color: $cod-gray;
+        }
+    }
+
+    &::before {
+        background: url(../images/triangle.png) repeat-x;
+        content: '';
+        height: 100%;
+        min-height: 12px;
+        min-width: 14px;
```

Comment:
No need for the min-width. And you can set the `height: 12px` instead of height-max-height it :smile: